src/commands/createWebApp/setDefaultRgAndPlanName.ts

```diff
@@ -18,22 +18,42 @@ const maxNumberOfSites: number = 3;
 export async function setDefaultRgAndPlanName(wizardContext: IAppServiceWizardContext, siteNameStep: SiteNameStep): Promise<void> {
     // this should always be set when in the basic creation scenario
     const location: Location = nonNullProp(wizardContext, 'location');
-    wizardContext.newResourceGroupName = `appsvc_${wizardContext.newSiteOS}_${location.name}`;
-    wizardContext.newPlanName = wizardContext.newResourceGroupName;
+    const defaultName: string = `appsvc_${wizardContext.newSiteOS}_${location.name}`;
 
-    const asp: AppServicePlan | null = await getAppServicePlan(wizardContext, wizardContext.newResourceGroupName, wizardContext.newPlanName);
-    if (asp) {
-        if (checkPlanForPerformanceDrop(asp)) {
-            // Subscriptions can only have 1 free tier Linux plan so show a warning if there are too many apps on the plan
-            if (wizardContext.newSiteOS === WebsiteOS.linux) {
-                await promptPerformanceWarning(wizardContext, asp);
-            } else {
-                // Subscriptions can have 10 free tier Windows plans so just create a new one with a suffixed name
-                // If there are 10 plans, it'll throw an error that directs them to advancedCreation
-                wizardContext.newResourceGroupName = await siteNameStep.getRelatedName(wizardContext, wizardContext.newPlanName);
-                wizardContext.newPlanName = wizardContext.newResourceGroupName;
+    const asp: AppServicePlan | null = await getAppServicePlan(wizardContext, defaultName, defaultName);
+    if (asp && checkPlanForPerformanceDrop(asp)) {
+        // Subscriptions can only have 1 free tier Linux plan so show a warning if there are too many apps on the plan
+        if (wizardContext.newSiteOS === WebsiteOS.linux) {
+            await promptPerformanceWarning(wizardContext, asp);
+            wizardContext.newResourceGroupName = defaultName;
```

Comment:
Would prefer if you did this:
```
wizardContext.newResourceGroupName = defaultName;
wizardContext.newPlanName = defaultName;
```

That way someone can't accidentally cause a bug by switching the order.